CONTRIBUTING.md

```diff
@@ -137,11 +138,13 @@ If successful, you should be able to see the results of your new theme under `te
 
 ### Update the `README.md` file
 
-Once you are happy with the colors and the rules in the style file, please update the README for your new theme!
+Once you are happy with the colors and the rules in the style file, please update the README for your new theme! You can generate a README with the following command:
 
-The **most** important part for us, is to add a table with the contrast ratios of the colors you've chosen and their compliance to WCAG. Please use any of the current themes as a base, and use any color contrast checker to fill it out.
+```py
+hatch dev:update_readmes --theme your-theme-name
```

Comment:
```suggestion
hatch run dev:update_readmes --theme your-theme-name
```

Reply:
I also couldn't not get jinja2 to autoinstall in the dev environment, and had to do a `hatch env run -e dev pip install jinja2`, I don't know if I'm just miss-using hatch.

Reply:
I see that you saw the line in #43 that addresses this